component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/x509Certificate/X509CertificateAuthenticator.java

```diff
@@ -517,5 +527,53 @@ private void addMatchStringsToList(Matcher matcher, Set<String> matches) {
         }
     }
 
+    private String getUserStoreDomainName(String userIdentifier, AuthenticationContext authenticationContext)
+            throws UserStoreException, AuthenticationFailedException {
+        if (Boolean.valueOf(getAuthenticatorConfig().getParameterMap().get(X509CertificateConstants.SEARCH_ALL_USERSTORES))) {
+            String[] filteredUsers = X509CertificateUtil.getUserRealm(userIdentifier).getUserStoreManager().listUsers
```

Comment:
Inorder to increase the readability lets assign X509CertificateUtil.getUserRealm(userIdentifier).getUserStoreManager() to a variable and use it here.

Reply:
Done